notes/lecture25.tex

```diff
@@ -0,0 +1,180 @@
+\section{Lecture 25: Interior Point Method}
+In last lecture, we have discussed about Newton's method. One advantage is its fast local convergence rate. However, there is no guarantee on global convergence for Newton's method. In this lecture, we'll present interior point method, which can be considered as an extension of Newton's method to ensure global convergence. First, the idea of interior point method will be introduced for general constrained optimization problems. Then the detailed algorithm and its convergence will be explained for linear programming. 
+\subsection{Barrier Methods}
+We'll firstly introduce barrier methods which convert the inequality constraints into a function added upon the objective function of a optimization problem. Consider the following optimization problem: 
+\begin{equation}
+\begin{aligned}
+&\min_x & {f(x)}\\
+&\qquad\text{s.t.} &x&\in{\domain},\\
+& &g_j(x)&\leqslant{0},\ j=1,2,\cdots,r,\\
+\end{aligned}
+\end{equation}
+where $f:\R^n\rightarrow\R$, $g_j:\R^n\rightarrow\R$ are given functions. $f$ is continuous, and $\domain$ is a closed set. For the rest of the lecture, we assume convex $g_j$ and $\domain=\R^n$. And we denote $x^*$ as the optimal solution of the problem.
```

Comment:
Don't start sentences with symbols, in this case $f$. Instead write "The function~$f$ is..."